app/api/domains/api.py

```diff
@@ -603,7 +604,8 @@ async def api_get_map_scores(
     # unlike /get_player_scores, we'll sort by score/pp depending
     # on the mode played, since we want to replicated leaderboards.
     if scope == "best":
-        sort = "pp" if mode >= GameMode.RELAX_OSU else "score"
+        if sort == None :
```

Comment:
vamos pythonizar! 

```python
if sort is None:
   ...
```

Reply:
Aiiiiiiiii to passando muito tempo em outras linguagens kkkkkkkkkkkkkk